src/Control/Lens/Lens.hs

```diff
@@ -143,11 +143,11 @@ import Control.Monad.State as State
 import Data.Functor.Apply
 import Data.Functor.Reverse
 import Data.Functor.Yoneda
-import Data.Monoid
 import Data.Profunctor
 import Data.Profunctor.Rep
 import Data.Profunctor.Sieve
 import Data.Profunctor.Unsafe
+import Data.Semigroup (Semigroup (..))
```

Comment:
Guard this behind `#if !(MIN_VERSION_base(4,11,0))` to avoid redundant import warnings on recent GHCs.